docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,133 @@
+membership, a corresponding Teleport user account will be created. If the Okta
+user already has a valid temporary Teleport SAML user account (i.e. they have
+logged into the cluster via SAML SSO before SCIM provisioning was enabled), the
+temporary account will automatically be adopted by the SAML integration and
+promoted to a long-lived SCIM-managed account.
+
+<Admonition type="note">
+Currently none of the SCIM user profile traits are stored in the Teleport user, although this may change in future.
+</Admonition>
+
+When a user is unassigned from the Okta App, or is disabled by the Okta Admin, Teleport will immediately delete the user in question, and create a lock that will both immediately terminate any existing sessions and prevent that user from re-using any credentials that have already been issued. This lock will be automatically revoked if the user is subsequently re-provisioned via SCIM, otherwise the lock is permanent and will have to be explicitly deleted to remove.
+
+<Admonition type="warning">
+Okta does not send SCIM updates to Teleport when a user is merely suspended. Even though Okta will prevent a suspended user from logging back into the cluster, any existing sessions will not be terminated, and any pre-issued credentials will be valid for their normal lifetimes.
+</Admonition>
+
+## Installing the Teleport SCIM integration
+
+Telepot support two SCIM integration modes:
+ * **Without API token** - User traits to role mapping will be propagated when the user logs in to Teleport.
+ As a side effect, user roles will be visible and updated when users log in to Teleport.
+ * **With API token** - User traits to role mapping will be propagated immediately during SCIM user provisioning.
+
+Run the following `tctl` command to install the SCIM integration.
+
+<Tabs>
+<TabItem label="Without API token">
+
+```bash
+$ tctl plugins install okta  \
+  --org https://trial-12356.okta.com \
+  --saml-connector "${SAML_CONNECTOR_NAME}" \
+  --no-users-sync \
+  --no-accesslist-sync \
+  --no-appgroup-sync \
+  --scim
+Successfully created OKTA plugin "okta"
+
+SCIM Base URL: https://teleport.example.com:443/v1/webapi/scim/okta
+SCIM Identifier field for users: userName
+SCIM Bearer Token: 1234567891234567891234567890
+```
+
+</TabItem>
+<TabItem label="With API token">
+
+```bash
+$ tctl plugins install okta  \
+  --org https://trial-12356.okta.com \
+  --saml-connector "${SAML_CONNECTOR_NAME}" \
+  --no-users-sync \
+  --no-accesslist-sync \
+  --no-appgroup-sync \
+  --scim
+  --api-token="${OKTA_API_TOKEN}"
+Successfully created OKTA plugin "okta"
+
+SCIM Base URL: https://teleport.example.com:443/v1/webapi/scim/okta
+SCIM Identifier field for users: userName
+SCIM Bearer Token: 1234567891234567891234567890
+```
+
+</TabItem>
+</Tabs>
+
+## Configuring the Okta app
+See the [Okta Integration *Configuring SCIM provisioning*](../hosted-guide.mdx#configuring-scim-provisioning) guide.
```

Comment:
- This link 404s for me. 
- To avoid breaking the user's flow, we should either (a) mention in the Prerequisites that a user must follow this guide before following the SCIM-only guide; or (b) explicitly state why the user should follow this guide and what objective they should complete before returning to this guide. Otherwise, the user needs to figure out a lot for themselves, which can be frustrating.